build.sh

```diff
@@ -134,9 +134,9 @@ then
     if [[ "$runcoverage" = true && -f "$testdir/coverage.xml" ]]
     then
       echo "(Running with coverage)"
-      (cd "$testdir"; $DOTCOVER cover "coverage.xml" /ReturnTargetExitCode)    
+      (cd "$testdir"; dotnet dotcover test -c Release --no-build --dcXML=coverage.xml)    
     else
-      dotnet test -c Release $testproject
+      dotnet test -c Release --no-build $testproject
```

Comment:
I changed this at the same time before I *believe* it will make the build faster. (We've just built everything, so we don't need to check for it.)